app/views/display_task/single_task.html.erb

```diff
@@ -3,18 +3,24 @@
     <th>Task Name</th>
     <th>Description</th>
     <th>Status</th>
-    <th>Completed date</th>
+    <th>Person in Charge</th>
+    <th>Completed Date</th>
   </tr>
 
   <tr>
     <td><%= @task.name %></td>
     <td><%= @task.description %></td>
-    <td><% if @task.completed_at.nil? %>
-          Not yet completed.
-        <% else %>
-          Completed.
-        <% end %>
+    <td>
+      <% if @task.completed_at.nil? %>
+        Not yet completed.
+      <% else %>
+        Completed.
+      <% end %>
     </td>
+    <td> <%= @person ? @person.name : "" %> </td>
```

Comment:
I think you could utilize the same ternary structure for the completed_at logic that you do above